src/foundry/client-esm/applications/api/dialog.d.mts

```diff
@@ -85,15 +82,12 @@ declare class DialogV2 extends ApplicationV2<EmptyObject, DialogV2.Configuration
    */
   protected _onSubmit(target: HTMLButtonElement, event: PointerEvent | SubmitEvent): Promise<DialogV2>;
 
-  protected override _onFirstRender(
-    _context: ApplicationV2.RenderContext,
-    _options: Partial<DialogV2.Configuration>,
-  ): void;
+  protected override _onFirstRender(_context: EmptyObject, _options: DeepPartial<ApplicationV2.RenderOptions>): void;
```

Comment:
This is all tracing from the first argument in the generics at the very top. Ultimately DialogV2 is *not* meant to be used with `_prepareContext` in part because there's no Handlebars support; everything dynamic here comes from `this.options` (the DialogV2.Configuration interface)